Math.ark

```diff
@@ -93,4 +93,40 @@
             (set divisors (append divisors i)))
         (set i (+ i 1))
     })
-    (append divisors n)}))
\ No newline at end of file
+    (append divisors n)}))
+
+###
+# @brief Returns the logarithm base n of a number
+# @param x the number
+# @param n the base
+# @author https://github.com/Gryfenfer97
+# = begin
+# (math:log 81 3) # Returns 4
+# = end
```

Comment:
I think that's `=begin` and `=end` (no space) otherwise the token won't get recognized by ArkDoc (same for the others)